templates/taxi/manufacturer_list.html

```diff
@@ -7,6 +8,10 @@ <h1>
       Create
     </a>
   </h1>
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" value="Search" class="btn btn-secondary">
+  </form>
```

Comment:
Duplicate code. Create additional template for this and then include it in places where you need this